### PR TITLE
added inherited, parent and descendant numbers to mutations df. fixes…

### PR DESCRIPTION
**Pull request by savitakartik**
… issue #38

### CODE REVIEW

**Comment by savitakartik** [comment]
image

File: tests/test_data_model.py [hunk 1]
```diff
@@ -19,6 +19,42 @@ def single_tree_example_ts():
         tables.mutations.add_row(site=j, derived_state="T", node=j)
     return tables.tree_sequence()
 
+def single_tree_recurrent_mutation_example_ts():
+    # 2.00 ┊                    6                    ┊
+    #      ┊            ┏━━━━━━━┻━━━━━━━┓            ┊
```
Comment (jeromekelleher) [inline tests/test_data_model.py:24]: Nice!

File: utils.py [hunk 2]
```diff
@@ -59,16 +61,77 @@ def mutations_data(self):
...
+        self.mutations_position = ts.sites_position[ts.mutations_site].astype(
+            int
+        )
+        N = ts.num_mutations
+        mutations_num_descendants = np.zeros(N, dtype=int)
+        mutations_num_inheritors = np.zeros(N, dtype=int)
+        mutations_num_parents = np.zeros(N, dtype=int)
+
+        tree = ts.first()
+        iterator = tqdm.tqdm(np.arange(N), desc="Classifying mutations")
```
Comment (jeromekelleher) [inline utils.py:94]: Let's get rid of the progress bar for now, it'll complicate things quite a bit. Can add later if we want it.

**Review by jeromekelleher — COMMENTED** [review]
Looks great! Just one comment really.

Looks like it's a bit out of sequence with some other PRs though? (We have some changes from #47 and others I think?)

**Review by jeromekelleher — APPROVED** [review]

**Pull request merged**